backend/Recodo.BLL/Exceptions/InvalidTokenException.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Recodo.BLL.Exceptions
+{
+    public class InvalidTokenException : Exception
+    {
+        public InvalidTokenException(string tokenName) : base($"Invalid {tokenName} token.") { }
```

Comment:
```suggestion
        public InvalidTokenException(string tokenName) 
            : base($"Invalid {tokenName} token.") 
        {
        
        }
```

The same in other places. We can do this at least at the beginning.